packages/platform-android/src/commands/runAndroid/index.ts

```diff
@@ -115,15 +109,18 @@ function getPackageNameWithSuffix(
 }
 
 // Builds the app and runs it on a connected emulator / device.
-function buildAndRun(args: Flags) {
-  process.chdir(path.join(args.root, 'android'));
-  const cmd = process.platform.startsWith('win') ? 'gradlew.bat' : './gradlew';
+function buildAndRun(args: Flags, config: Config) {
+  const androidConfig = config.project.android;
```

Comment:
Didn't we check for existence already? If it's guaranteed to be there, maybe we can pass "config.project.android" directly or use some special Typescript utility type?